src/garage/torch/policies/gaussian_mlp_policy.py

```diff
@@ -13,15 +13,17 @@ class GaussianMLPPolicy(Policy, GaussianMLPModule):
 
     Args:
         env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
-        kwargs : Additional keyword arguments passed to the GaussianMLPModule.
+        kwargs (dict): Additional keyword arguments passed to the
+            GaussianMLPModule.
+        name (str): Name of policy.
 
     """
 
-    def __init__(self, env_spec, **kwargs):
+    def __init__(self, env_spec, name='GaussianMLPPolicy', **kwargs):
```

Comment:
please don't swallow `**kwargs` in the constructor. (I know you didn't write this, but let's remove it)

Reply:
If we remove it then how can one change the non-linearility of layer or initialization of weights?, or the point hear is that no one should be changing that through `GaussianMLPPolicy`?

Reply:
I see what you mean. We should explicitly mention each keyword argument that this class supports?

Reply:
yes. always.